pkgs/top-level/all-packages.nix

```diff
@@ -9580,15 +9580,22 @@ in
 
   rust_1_44 = callPackage ../development/compilers/rust/1_44.nix {
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
+    llvmPackages = llvmPackages_9;
   };
   rust_1_45 = callPackage ../development/compilers/rust/1_45.nix {
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
+    llvmPackages = llvmPackages_10;
+  };
+  rust_1_46 = callPackage ../development/compilers/rust/1_46.nix {
+    inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
+    llvmPackages = llvmPackages_10;
   };
-  rust = rust_1_45;
+  rust = rust_1_46;
```

Comment:
Someone should check what version we use for building firefox right now and if this is affected by this change.

Reply:
:+1: 
ofborg might have done a build already but I'm not too familiar with where to look for that
https://github.com/NixOS/nixpkgs/pull/96890#issuecomment-684509529

I don't plan on merging this until `rust_1_46` is in master either as default or as a separate optional version.

#96890